src/patternfly/components/Card/card.scss

```diff
@@ -125,6 +126,10 @@
     }
   }
 
+  &.pf-m-fill {
```

Comment:
It doesn't seem like this is a requirement for the PR. Also looks like you're using `pf-u-h-100` in a handful of other places. I'm curious if we need this, or that use of the utility classes, or if we could make the demo designs more simple since as far as I can tell the use of `pf-m-fill` and the utility classes aren't essential to the tab layout in any way.

@mcarrano wdyt? This adds a variation to the card component that tells it to fill the available height since some of the cards in these demos fill the space like that. And then we're using utility classes in place of custom CSS for some other parts of the layout, like making these boxes below take up a bunch of vertical height using a height utility class since they don't have any content and don't take up height naturally.

Generally we avoid using utility classes in the demos since utility classes don't come with PF by default, and can be a heavy import to include them (which is why we don't include them by default), so we don't want to make it seem like a feature relies upon them being included.

I'm curious if it would be OK to alter the demo design a little so we don't have to use the utilities and add a new card variation. It would mean some of the boxes don't fill all of the height the way they do now, and maybe we don't use a grid layout in a layout like below and just remove the cards on the right.

<img width="2007" alt="Screen Shot 2021-03-10 at 5 13 19 PM" src="https://user-images.githubusercontent.com/35148959/110710728-02792900-81c4-11eb-8234-6e6a3cff6bda.png">

Reply:
Perhaps try to simplify this and we can take a look at it. I understand the issue and agree that this demo should focus on the tabs. Other elements on the page are not so relevant as long as the final result looks presentable.

Reply:
`pf-m-fill` is necessary for cards inside layout `__item` s as they don't fill the height of their containers. I used it here in the demo context.

Reply:
We could also put content in either of the cards on the right to give them height (or just remove them), and put content in the tab content to give it height, since to make them as tall as they are here with no/empty content requires a new variation to the card.

Reply:
This is the sitch I'm describing. We need a `full height` option if using layout `item` wrappers. Although they aren't necessary (which we've noted), to make cards full height if you are using them, say for `Gallery` or `Grid`:

Before 

![Screen Shot 2021-03-12 at 9 48 06 AM](https://user-images.githubusercontent.com/5385435/110956580-c4414e00-8318-11eb-989f-71b063df8a61.png)

After

![Screen Shot 2021-03-12 at 9 52 32 AM](https://user-images.githubusercontent.com/5385435/110956885-171b0580-8319-11eb-8ad7-a77678ae0edf.png)

Before

![Screen Shot 2021-03-12 at 9 51 29 AM](https://user-images.githubusercontent.com/5385435/110957280-76791580-8319-11eb-955f-7f9177c27695.png)

After

![Screen Shot 2021-03-12 at 9 51 46 AM](https://user-images.githubusercontent.com/5385435/110957834-061ec400-831a-11eb-8d5b-d89b59a37c4f.png)

Before:
 This is the most pertinent example. **`pf-l-grid__item` is necessary to span grid columns.In that case, the card has no way of consuming full height of its parent, unless using a `card modifier` or a `utility class / pf-u-h-100`.**

![Screen Shot 2021-03-12 at 9 51 16 AM](https://user-images.githubusercontent.com/5385435/110957893-1636a380-831a-11eb-8c7c-6adf3db445fe.png)

After

![Screen Shot 2021-03-12 at 9 52 03 AM](https://user-images.githubusercontent.com/5385435/110957863-0dde6880-831a-11eb-8ec3-44026233e104.png)